File: math-functions/c/esempio.c
//Per utilizzare queste funzioni in un altro programma, è sufficiente includere l'intestazione della libreria e richiamare le funzioni come se fossero funzioni del programma stesso. 
//Ad esempio:

#include "math_functions.h"

int main(int argc, char** argv) {
  double x = 2.0;
  double y = 3.0;

  printf("Radice quadrata di %f: %f\n", x, sqrt(x));
  printf("Potenza di %f elevato a %f: %f\n", x, y, pow(x, y));
  printf("Logaritmo di %f: %f\n", x, log(x));

  return 0;

File: math-functions/rust/esempio.rs
//Per utilizzare questa libreria, puoi importarla nel tuo codice e chiamare le sue funzioni come segue:





use math::{sqrt, pow, log, cube, circle_area};

fn main() {
    let x = 2.0;
    let y = 3.0;
    let r = 1.0;

    println!("Radice quadrata di {}: {}", x, sqrt(x));
    println!("Potenza di {} elevato a {}: {}", x, y, pow(x, y));
    println!("Logaritmo di {}: {}", x, log(x));
    println!("Cubo di {}: {}", x, cube(x));
    println!("Area di un cerchio di raggio {}: {}", r, circle_area(r));
}
